src/seth/libexec/seth/seth-estimate

```diff
@@ -15,7 +16,7 @@
 ### With `-F <sender>', simulate calling <receiver> from <sender>
 ### With `-V <value>', simulate transferring <value> to <receiver>.
 set -e
-[[ $2 ]] || seth --fail-usage "$0"
+[[ $1 ]] || seth --fail-usage "$0"
```

Comment:
why?

Reply:
I added a new option for `seth estimate` which allows the form `seth estimate --create <data>`. This is useful in our context when we want to estimate the cost of a transaction on a much lower level after everything has already been encoded. Had I not modified the above command, it would have failed whenever using this new form since it only has one argument.